core/src/main/scala/org/http4s/headers/Content-Length.scala

```diff
@@ -9,7 +9,7 @@ import org.http4s.util.Writer
   *
   * The HTTP RFCs do not specify a maximum length.  We have decided that `Long.MaxValue`
   * bytes ought to be good enough for anybody in order to avoid the irritations of `BigInt`.
-  *t
+  * 
   * @param length the length; throws an `IllegalArgumentException` if negative
```

Comment:
Is this still true? We have been removing exception throwing form the constructors

Reply:
It doesn't seem true. Only `unsafeFromLong` seems to be throwing.